Makefile.joy

```diff
@@ -76,6 +76,11 @@ joy_api_test:
 	@if [ ! -d "lib" ]; then mkdir lib; fi;
 	@cd src; $(MAKE) $(MAKEFLAGS) joy_api_test
 
+joy_memory_profile:
```

Comment:
So Makefile.joy is really only temporary. I kept it around in case something didn't work with autotools.

Reply:
I am not really sure I even want to keep this tool around. I committed it just because, but it might not be necessary. Understood about Makefile.joy. I noticed I couldn't build this particular tool with auto tools, I have to use Makefile.joy. Again, not sure I even want to keep it around anyways.

Reply:
Ok. If you want to add it, I can do that. Let me know.